blogs/lokiless_netobserv/index.md

```diff
@@ -0,0 +1,138 @@
+
+# Configure Network Observability without Loki
+Loki as datasource is currently enabled by default. To configure Network Observability operator without Loki, when configuring Flowcollector resource simply set `.spec.loki.enable` to `false`
+
+```yaml
+loki:
+  enable: false
+```
+When configured as above, Network Observability's Prometheus metrics will continue to get scraped by OpenShift's cluster Prometheus without any additonal configuration and Network Traffic console will use Prometheus as a source for fetching the data.
+
+## Performance and Resource utilization gains
+
+### Query performance:
+Prometheus queries are blazing fast compared to Loki queries, but don't take my word for it, let's look at the data from the query performance tests: 
+
+Test bench environment:
+
+* Test: We did identical 50 queries for 3 separate time ranges to render topology view for both Loki and Prometheus. Such query requests all K8s Owners for the workload running in an OpenShift Cluster that had  network flows associated to them. Since we didn't have any applications running, it was all Infrastructure workloads generating network traffic. In Network Observability such unfiltered view will have topology rendered as below: 
+
+    ![unfiltered topology view](images/owner_screenshot.png)
+
+* Test bed: 9 worker and 3 master nodes, AWS m5.2xlarge machines
+* LokiStack size: 1x.small
+
+Results:
+
+  Below table shows 90th Percentile query times for each table:
+
+  | Time Range | Loki      | Prometheus
+  | :--------: | :-------: | :----------:
+  | Last 5m    | 984 ms    | 99 ms
+  | Last 1h    | 2410 ms   | 236 ms
+  | Last 6h    | > 10 s    | 474 ms
+
+As time range to fetch network flows gets wider, Loki queries tends to get slower or timing out, while Prometheus queries is able render the data within fraction of a second.
+
+### Resource utilization:
+In our tests conducted on 3 different test beds with varied workloads and network throughput, when Network Observability is configured without Loki, total savings of Memory usage could be in range 45-65% and CPU utilzation could be lower by 10-20%<sup>*</sup>. Not to mention you will not need to provision and plan for additional object storage in public clouds for Loki, overall reducing the cost and improving operational efficiency significantly.
+
+In our perf tests, [kube-burner](https://github.com/kube-burner/kube-burner) workloads were used to generate several objects and create heavy network traffic. We used sampling rate of 1 for all below tests. To further describe each test bed:
```

Comment:
```suggestion
In our perf tests, [kube-burner](https://github.com/kube-burner/kube-burner) workloads were used to generate several objects and create heavy network traffic. We used a sampling rate of 1 for all of the following tests:
```

Reply:
suggest replacing "below" with "following" and IMO, you don't need to have the "To further describe bit". Just go ahead with the colon :)